packages/front-end/components/Settings/SnowflakeForm.tsx

```diff
@@ -1,12 +1,15 @@
-import { FC, ChangeEventHandler } from "react";
+import { FC, ChangeEventHandler, useState } from "react";
 import { SnowflakeConnectionParams } from "back-end/types/integrations/snowflake";
 import Tooltip from "@/components/Tooltip/Tooltip";
+import Toggle from "@/components/Forms/Toggle";
 
 const SnowflakeForm: FC<{
   params: Partial<SnowflakeConnectionParams>;
   existing: boolean;
   onParamChange: ChangeEventHandler<HTMLInputElement>;
-}> = ({ params, existing, onParamChange }) => {
+  onManualParamChange: (name: string, value: string) => void;
+}> = ({ params, existing, onParamChange, onManualParamChange }) => {
+  const [useAccessUrl, setUseAccessUrl] = useState(false);
```

Comment:
Should the default state value be set to `true` if the existing params already has a non-empty `accessUrl`?  Thinking about editing a data source - if they were using an accessUrl, we want that section to be expanded already when they go to edit.

Reply:
correct, fixed.